contxt/cli/commands/ems.py

```diff
@@ -28,43 +28,43 @@ def ems() -> None:
 @ems.command()
 @click.argument("facility_id")
 @click.option("--resource-type", type=ResourceType, default="electric", help="Resource type")
-@fields_option(default="id, name, resource_type", obj=MainService)
-@sort_option(default="id")
+# TODO fix field selection and sorting
```

Comment:
the projection of main services was configurable, and the results were sortable. I need to make that work again if that is indeed useful